bytecode/instr.py

```diff
@@ -146,12 +146,10 @@ def _check_arg_int(name, arg):
 
 
 _stack_effects = {
```

Comment:
I've done something different that what you suggested.  Since converting to True/False/None already trashed the jump-free indexing of the 3-tuple, I just converted the whole thing to a 2-tuple and using max() at runtime.